deploy/helm/templates/_common-discovery-config.tpl

```diff
@@ -481,8 +498,18 @@ metrics/relationship-state-events-workload-service-preparation:
     - groupbyattrs/istio-relationships
     - transform/only-relationship-resource-attributes
   exporters:
+    - forward/discovery-istio-metrics-clean
     - solarwindsentity/istio-workload-service
 
+metrics/discovery-istio-clean:
```

Comment:
Istio metrics that do not have attributes passing `filter/keep-workload-workload-relationships` or `filter/keep-workload-service-relationships` won't reach this pipeline. Can all metrics pass at least one of the filters?

Reply:
should be fixed